data/cron.py

```diff
@@ -0,0 +1,68 @@
+import json
+from collections import defaultdict
+from time import sleep
+
+import requests
+from IGitt.GitHub import GitHub
+
+from coala_web.settings import GITHUB_API_KEY
+from org.models import Contributor
+
+request_headers = {'Authorization': 'token %s' % GITHUB_API_KEY}
+
+
+def fetch_org():
+    data = defaultdict(lambda: defaultdict(int, {'name': '', 'bio': ''}))
+    repo_url = '/orgs/coala/repos'
+    response_repo = GitHub.get(GITHUB_API_KEY, repo_url)
+    for repo in response_repo:
+        commit_data = get_commits(repo, data)
+        data = get_issues(repo, commit_data)
+
+    for contributor in data:
+        response = \
+            requests.get('https://api.github.com/search/issues?q=commenter:' +
+                         contributor + '%20is:pr%20user:coala',
+                         headers=request_headers, timeout=3000)
+        r_json = json.loads(response.text)
+        data[contributor]['reviews'] = r_json['total_count']
+        if int(response.headers['X-RateLimit-Remaining']) == 1:
```

Comment:
Can't we simply keep it string comparison?

Reply:
fyi, this is irrelevant; the PR isnt doing what the issue requests.  This entire module must be removed.